### ISSUE TITLE
Centaur-tabs doesn't recolor itself when I change bright or dark themes

### ISSUE THREAD

**Issue by hwiorn** [doomemacs/doomemacs#6503]
### What did you expect to happen?

Default theme of doom-emacs theme is `doom-one`.
When I change a light theme like `doom-light` or `tdsh-light`, centaur-tabs should change their color scheme. 

### What actually happened?

When I change a light theme like `doom-light` or `tdsh-light`, centaur-tabs doesn't redraw itself.
When with no AC power, I prefer light themes on the go because I can recognize easily low brightness display.
It's a little annoying I can't see the tab-bar with low brightness display mode.
I tried to find a centaurs-tab's frame redrawing code, But I couldn't find it.

### Describe your attempts to resolve the issue

The only solution I found is change the `doom-theme` and press `SPC h r r`, then restart Emacs.

### Steps to reproduce

1. Enable `:ui tabs`(centaur-tabs) feature
2. Open Emacs with default dark or light theme(e.g. `doom-one`)
3. Open a project and some files to make tabs.
4. Change theme to lighter, vice versa.

### System Information

https://pastebin.com/d7297PtH

**Comment by hlissner** [comment]
As of ea1f72e, centaur-tabs-mode is reset if the theme is changed. Thanks for bringing it to my attention!

**Comment by hwiorn** [comment]
@hlissner I had almost forgotten about this issue as well. Unfortunately, the commit you applied does not work. Simply running `M-x centaur-tabs-mode` does not reapply the current theme. I encountered this problem in 2022, and while checking centaur-mode, I had to stop due to lack of time. The same issue can be reproduced by cycling through the themes like this:

doom-dark -> doom-one-light -> tsdh-dark -> tsdh-light -> ... (repeat)

Check this: https://github.com/ema2159/centaur-tabs/issues/157
